README.md

```diff
@@ -701,10 +705,381 @@ class Calendar(component.Component):
         return {
             "date": kwargs["my-date"],
             "id": kwargs["#some_id"],
-            "@click.native": kwargs["@click.native"]
+            "on_click": kwargs["@click.native"]
         }
 ```
 
+### Pass dictonary by its key-value pairs
+
+_New in version 0.74_:
+
+Sometimes, a component may expect a dictionary as one of its inputs.
+
+Most commonly, this happens when a component accepts a dictionary
+of HTML attributes (usually called `attrs`) to pass to the underlying template.
+
+In such cases, we may want to define some HTML attributes statically, and other dynamically.
+But for that, we need to define this dictionary on Python side:
+
+```py
+@component.register("my_comp")
+class MyComp(component.Component):
+    template = """
+        {% component "other" attrs=attrs %}
+        {% endcomponent %}
+    """
+
+    def get_context_data(self, some_id: str):
+        attrs = {
+            "class": "pa-4 flex",
+            "data-some-id": some_id,
+            "@click.stop": "onClickHandler",
+        }
+        return {"attrs": attrs}
+```
+
+But as you can see in the case above, the event handler `@click.stop` and styling `pa-4 flex`
+are disconnected from the template. If the component grew in size and we moved the HTML
+to a separate file, we would have hard time reasoning about the component's template.
+
+Luckily, there's a better way.
+
+When we want to pass a dictionary to a component, we can define individual key-value pairs
+as component kwargs, so we can keep all the relevant information in the template. For that,
+we prefix the key with the name of the dict and `:`. So key `class` of input `attrs` becomes
+`attrs:class`. And our example becomes:
+
+```py
+@component.register("my_comp")
+class MyComp(component.Component):
+    template = """
+        {% component "other"
+            attrs:class="pa-4 flex"
+            attrs:data-some-id=some_id
+            attrs:@click.stop="onClickHandler"
+        %}
+        {% endcomponent %}
+    """
+
+    def get_context_data(self, some_id: str):
+        return {"some_id": some_id}
```

Comment:
You are right, and the example does work that way. The `get_context_data` is for component `my_comp`, where we don't need to specify anything, because it's the parent. If we've shown also code for component `other`, then `other`'s `get_context_data` must include `attrs` among the inputs.

I can add a dummy code for the `other` component to avoid the confusion

Reply:
Oh, my fault! No need to add anything else then.